### PR TITLE
adding ParseMax w/ max form size in signature - forwarded to from Parse

### LINKED ISSUE #3: feature: add maximum data size to Parse signature
currently when parsing `multipart/form-data` requests, the size of data is locked in at [2048](https://github.com/albrow/forms/blob/master/data.go#L57). For image uploads this might be a little low; is it possible to add the maximum size to the functions signature? Or perhaps a `ParseMax(*http.Request, uint)` function?

### PR DESCRIPTION
**Pull request by dadleyy**
closes #3

### CODE REVIEW

File: data.go [hunk 1]
```diff
@@ -15,6 +15,8 @@ import (
 	"strings"
 )
 
+const DefaultMaxFormSize = 2048
```
Comment (albrow) [inline data.go]: I like for all exported identifiers to have doc comments. Mind adding a comment here? It doesn't have to be long. Just something like:

```
// DefaultMaxFormSize is the default maximum form size (in bytes) used by the Parse function.
```

Also, I know this is my fault, but I think 2048 is probably much too small. I want to find a value that is likely to be safe (i.e. is unlikely to cause OOM exceptions in most environments) and allows for common things like text files, pdfs, and smaller images. Shall we try something around 500 KB?

**Review by albrow — CHANGES_REQUESTED** [review]
I just left one small comment. After it's addressed and I've confirmed the tests pass, I'll merge.

File: data.go (hunk 1 above)
Comment (dadleyy) [reply data.go]: Sounds good to me. I'll update in a bit! Thanks

**Comment by dadleyy** [comment]
@albrow just made your changes. I also added a [`.travis.yaml`](https://github.com/dadleyy/forms/blob/master/.travis.yml) file. It [runs the tests](https://travis-ci.org/dadleyy/forms).

**Comment by albrow** [comment]
@dadleyy thanks for updating this PR. I'm sorry for the delay, but my main development computer has been out of commission for a while. I'll likely be up and running again in the next week or so and I can do a more thorough review at that time.

> I also added a .travis.yaml file.

Thank you for this :) I have nothing against Travis but I've been using CircleCI for many of my other projects  ([albrow/zoom](https://github.com/albrow/zoom) for example) and I'm more familiar with it. I'll leave __.travis.yaml__ there for now but I'll likely remove it after I set up CircleCI.

**Review by albrow — APPROVED** [review]

**Comment by albrow** [comment]
Thanks again, @dadleyy! I merged this and released version 0.4.0.

**Pull request merged**